scope_extensions-explainer.md

```diff
@@ -1,126 +1,216 @@
+could confuse users. 
+
+### Out-of-scope navigations leave the app window
+
+When a page in an app window navigates to an out-of-scope URL, some browser
+implementations may handle it by opening a new tab or browsing context in a
+regular browser window. This can appear jarring for users if they are unfamiliar
+with how scope works and the content appears to still be part of the app.
+Developers can use `scope_extensions` to ensure appropriate navigations do not
+leave the app window. 
+
+### Navigation Capture (a.k.a. In-browser Link Capture)
+
+Navigation capturing is a behavior that captures suitable browser window page
+navigations and opening them an app window when the target URL matches the scope
+of an installed app. This behavior is common on mobile platforms and is [in
+development](https://issues.chromium.org/issues/40058251) for desktop platforms
+in Chromium. Navigation capturing relies on the scope of an app to decide if a
+navigation is entering or leaving an app. Developers can use `scope_extensions`
+to ensure appropriate browser navigations open in an app window. 
+
+Navigation capturing can be combined with
+[`launch_handlers`](https://developer.mozilla.org/en-US/docs/Web/Manifest/launch_handler)
+to create a tailored app navigation experience.  
+
+### Other manifest features
+As app scope is used as the boundary for the application of manifest metadata
+such as
+[`theme_color`](https://developer.mozilla.org/en-US/docs/Web/Manifest/theme_color),
+documents intended to be part of the app but are excluded from app scope cannot
+take part and are not presented in a consistent manner. [Window Controls
+Overlay](https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API)
+is another example - the overlay is not applied to the titlebar when the main
+document in the app window is out-of-scope. 
 
-![Installed web app showing out-of-scope UX](images/out-of-scope-ux.png)
+## Goals
 
-The app in the image (`PWinter`) has navigated to a url out of its scope
-(`airhorner.com`). The white bar on top of the web app is providing information
-to the user about this change of scope. While this is a security feature, it can
-be the case that an application wants to extend its scope. For example, an
-application might host content that is located in one specific origin, and rely
-on a login page that is out of the scope of the application itself to access
-that content. In other cases, the same application might be associated to
-multiple Top Level Domains, that might respond to different locales
-(`app.com`, `app.co.uk`, `app.co.cr`, etc).
+- Allow developers to define an app scope using a list of origins known at
+  install time.
 
+- Allow developers to specify a scoping path for each listed origin similar to
+  how the manifest `scope` field works.
+
+- Allow web apps to capture user navigations to sites they are affiliated with.
+  E.g. "News Aggregator App" capturing links navigations to examplenewssite.com.
```

Comment:
This scenario is very real, but this specific example seems weak. It feels less likely that a news site would establish the relationship with an aggregator.  What about a company that has a suite of apps on different origins, but would like them all to feel connected within a single uber-app.

Reply:
Done. Replaced with a productivity suite example.